core/src/main/scala/com/pennsieve/db/DatasetReleaseTable.scala

```diff
@@ -42,6 +42,8 @@ final class DatasetReleaseTable(schema: String, tag: Tag)
   def tags = column[List[String]]("tags")
   def createdAt = column[ZonedDateTime]("created_at", O.AutoInc)
   def updatedAt = column[ZonedDateTime]("updated_at", O.AutoInc)
+  def releaseStatus = column[String]("release_status")
+  def publishingStatus = column[String]("publishing_status")
 
```

Comment:
Is it worth the trouble to have these be `column[DatasetReleaseStatus]` and `column[DatasetReleasePublishingStatus]`? I think it involves adding something to the PostgresProfile?

Reply:
Yes, it is. I think that I missed this because I added the columns and the case classes at different times. I will make this revision.

Reply:
Okay, thanks!